graphdatascience/query_runner/neo4j_query_runner.py

```diff
@@ -161,20 +161,28 @@ def run_cypher(
         query: str,
         params: Optional[dict[str, Any]] = None,
         database: Optional[str] = None,
+        mode: Optional[QueryMode] = None,
         custom_error: bool = True,
         connectivity_retry_config: Optional[ConnectivityRetriesConfig] = None,
     ) -> DataFrame:
         if params is None:
             params = {}
 
+        if mode is None:
+            mode = QueryMode.WRITE
```

Comment:
just curious: why do the defaulting here instead of in the parameter declaration?